cmd/kured/main.go

```diff
@@ -566,7 +626,14 @@ func rebootAsRequired(nodeID string, rebootCommand []string, sentinelCommand []s
 			continue
 		}
 
+		enableExcludeFromELBs(client, nodeID)
 		drain(client, node)
+
+		if rebootDelay > 0 {
```

Comment:
Next to that, I would probably use the existing throttle function, maybe rename and refactor it so that it's clearer.

I am worried the users might not understand the difference between all those options we have, tbh, so I think it might be worth clarifying in the CLI arguments.